manifests/0000_90_kube-apiserver-operator_04_servicemonitor-apiserver.yaml

```diff
@@ -106,6 +106,24 @@ metadata:
     exclude.release.openshift.io/internal-openshift-hosted: "true"
 spec:
   groups:
+  - name: aggregated-apis
+    rules:
+    - alert: AggregatedAPIErrors
+      annotations:
+        message: "An aggregated API {{ $labels.name }}/{{ $labels.namespace }} have reported errors."
+        description: "The number of errors have increased for an aggregated API {{ $labels.name }}/{{ $labels.namespace }} in the last minute. Note that high values indicate that the availability of the services changes too often."
+      expr: |
+        sum by(name, namespace)(increase(aggregator_unavailable_apiservice_count[1m])) > 2
+      labels:
+        severity: warning
```

Comment:
@sichvoge we are kind of blindly impementing these alerts, without having some rule of thumbs about severity and how quick they should fire. Can you comment on this, here in this case about aggregated APIs being down for one minute (like service-catalog, openshift-apiserver or metrics apiserver) ?

Reply:
5 minute is more of a normal time range, so I would change it to that, for both of them.

I like to ask myself what can a user do if this fires, what would the rulebook/playbook say. How can they fix this?